include/libopencm3/swm050/clk.h

```diff
@@ -0,0 +1,55 @@
+/** @defgroup clk_defines Clock Defines
+ *
+ * @brief <b>Defined Constants and Types for the SWM050 System Clock</b>
+ *
+ * @ingroup SWM050_defines
+ *
+ * LGPL License Terms @ref lgpl_license
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2019 Caleb Szalacinski <contact@skiboy.net>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/**@{*/
+#ifndef LIBOPENCM3_CLK_H
+#define LIBOPENCM3_CLK_H
+#include <libopencm3/cm3/common.h>
+#include <libopencm3/swm050/memorymap.h>
+
+/* Various Clock Defines */
+/** @defgroup clk_misc_defs Various Clock Defines
+
+@{*/
+#define DEFAULT_BOOT_FREQ	(9000000U)
```

Comment:
what is this used for? in other families, that have used a global to store the clock speed, it just gets set to that, and edited, this define doesn't really seem terribly useful

Reply:
That's the default boot frequency according to the datasheet.  I can see why it could be removed, though.

Reply:
Fixed